src/hb-blob.cc

```diff
@@ -499,6 +499,43 @@ hb_blob_t::try_make_writable (void)
 # define MAP_NORESERVE 0
 #endif
 
+static hb_blob_t *
+_fread_to_end (FILE *file)
```

Comment:
I learned from https://codereview.stackexchange.com/q/9421 for this but hardly there is any connection between the two IMO.